scriptmodules/supplementary/skyscraper.sh

```diff
@@ -199,44 +205,33 @@ function configure_skyscraper() {
 function _init_config_skyscraper() {
     local scraper_conf_dir="$configdir/all/skyscraper"
 
-    # Make sure the `artwork.xml` and other conf file(s) are present, but don't overwrite them on upgrades
-    local f_conf
-    for f_conf in artwork.xml aliasMap.csv; do
-        if [[ -f "$scraper_conf_dir/$f_conf" ]]; then
-            cp -f "$md_inst/$f_conf" "$scraper_conf_dir/$f_conf.default"
-        else
-            cp "$md_inst/$f_conf" "$scraper_conf_dir"
-        fi
-    done
-
     # If we don't have a previous config.ini file, copy the example one
-    if [[ ! -f "$scraper_conf_dir/config.ini" ]]; then
-        cp "$md_inst/config.ini.example" "$scraper_conf_dir/config.ini"
-        sed -i 's/\[esgamelist\]/[esgamelist]\ncacheScreenshots="false"/' "$scraper_conf_dir/config.ini"
-    fi
+    [[ ! -f "$scraper_conf_dir/config.ini" ]] && cp "$md_inst/config.ini.example" "$scraper_conf_dir/config.ini"
 
     # Try to find the rest of the necessary files from the qmake build file
-    # They should be listed in the `unix:examples.file` configuration line
-    if [[ $(grep unix:examples.files "$md_build/skyscraper.pro" 2>/dev/null | cut -d= -f2-) ]]; then
-        local files=$(grep unix:examples.files "$md_build/skyscraper.pro" | cut -d= -f2-)
-        local file
-
-        for file in $files; do
-            # Copy the files to the configuration folder. Skip config.ini, artwork.xml and aliasMap.csv
-            if [[ $file != "artwork.xml" && $file != "config.ini" && $file != "aliasMap.csv" ]]; then
-                cp -f "$md_build/$file" "$scraper_conf_dir"
-            fi
-        done
-    else
-        # Fallback to the known resource files list
-        cp -f "$md_inst/artwork.xml.example"* "$scraper_conf_dir"
-
-        # Copy resources and readme
-        local resource_file
-        for resource_file in README.md mameMap.csv tgdb_developers.json tgdb_publishers.json hints.txt; do
-            cp -f "$md_inst/$resource_file" "$scraper_conf_dir"
-        done
+    # They should be listed in the `unix:examples.files` configuration line
+    iniConfig "=" "" "$md_build/skyscraper.pro"
+    iniGet "unix:examples.files"
+    local files="$ini_value"
+    if [[ -z "$files" ]]; then
+        files="README.md hints.xml artwork.xml artwork.xml.example1 artwork.xml.example2"
+        files+=" artwork.xml.example3 artwork.xml.example4 aliasMap.csv mameMap.csv"
+        files+=" docs/ARTWORK.md tgdb_developers.json tgdb_publishers.json platforms.json"
+        files+=" screenscraper.json mobygames.json"
     fi
+    local file
+    for file in $files; do
+        case "$file" in
+            config.ini)
+                ;;
+            artwork.xml|aliasMap.csv|platforms.json|screenscraper.json|mobygames.json)
```

Comment:
IMHO only `artwork.xml` and `platforms.json` should be copied-if-not-found here, since they're the only ones meant for user edits.

Reply:
> since they're the only ones meant for user edits.

`screenscraper.json` does takes user edits, which are needed to scrape that module for the custom systems (e.g. **tg16**) that are added to `platforms.json`.

I don't use [aliasMap.csv](https://github.com/Gemba/skyscraper/blob/dev_reloaded/aliasMap.csv) (it [don't work with screenscraper](https://retropie.org.uk/forum/topic/31281/skyscraper-determining-search-name-for-game-entry) and that's my primary source), but it does seem meant for user edits also.

I only wasn't sure about `mobygames.json` but looking inside, it doesn't really seem like there's anything for us to do in that one so I'll remove that one here.